### PR TITLE
Learn page: map getting started content, add layout

### PR DESCRIPTION
**Pull request by c0ffeeca7**
## Proposed change

- Learn page: map getting started content
- Delete dummy getting started page
- Define _page_ as default layout
- Add side TOC to page 
- Add getting started CSS from old website

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_layouts/page.html [hunk 1]
```diff
@@ -2,6 +2,13 @@
 layout: default
 ---
 
+{% if page.toc %}
+{% include guides.html %}
+ 
+</script>
```
Comment (frenck) [inline source/_layouts/page.html]: Why is this script dangling here and not part of the `guides.html` include?

File: source/_layouts/page.html [hunk 2]
```diff
@@ -18,8 +25,10 @@ <h1 class="text-black text-[24px] md:text-[44px] font-extrabold tracking-[-0.02e
                 {{ page.title }}
             </h1>
             <div class="content post-content">
-                {{ page.content | markdownify | output_modder }}
+                {{ content }}
```
Comment (frenck) [inline source/_layouts/page.html]: Hmm, this removes the output handling we have for handling things like automatically adding `rel='external nofollow'` to outgoing links.

https://github.com/home-assistant/home-assistant.io/blob/982dfd1e9e43f072b79900e9d8d4f1adc40b6371/plugins/output_modder.rb#L6

Is there a specific reason this was removed?

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_layouts/page.html (hunk 2 above)
Comment (c0ffeeca7) [reply source/_layouts/page.html]: The reason this was removed was that some elements such as the includes were not displayed correctly:
image

But it turns out that this was caused by `page.content`. I've added the output_modder and markdownify back in.

File: source/_layouts/page.html (hunk 1 above)
Comment (c0ffeeca7) [reply source/_layouts/page.html]: no, I think that was a copy paste error.

File: source/_layouts/page.html (hunk 2 above)
Comment (frenck) [reply source/_layouts/page.html]: 👍 

Nice!

File: source/_includes/guides.html [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+                            li.classList.add('guide-active');
+                        } else {
+                            li.classList.remove('guide-active');
+                        }
+                        
+                    }
+                }
+            });
+        });
+    });
```
Comment (frenck) [inline source/_includes/guides.html:76]:
```suggestion
    });
</script>
```

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — APPROVED** [review]
Looking good! 👍 

../Frenck

**Pull request merged**